src/components/dialogs/parameters/security-analysis-parameters.js

```diff
@@ -36,9 +259,40 @@ export const SecurityAnalysisParameters = ({
                 <DropDown
                     value={provider}
                     label="Provider"
-                    values={providers}
+                    values={securityAnalysisiProvider}
                     callback={updateProviderCallback}
                 />
+
+                <Grid className={classes.textContainer}>
+                    <Grid item xs={8} className={classes.text}>
+                        <Typography>
+                            {intl.formatMessage({
+                                id: 'securityAnalysis.violationsHiding',
+                            })}
+                        </Typography>
+                        <Tooltip
+                            className={classes.tooltip}
+                            title={
+                                <div
+                                    dangerouslySetInnerHTML={{
+                                        __html: intl.formatMessage({
+                                            id: 'securityAnalysis.toolTip',
+                                        }),
+                                    }}
+                                />
+                            }
+                            placement="left-start"
+                        >
+                            <InfoIcon />
+                        </Tooltip>
```

Comment:
maybe it's simplier (and less "dangerous") to use :

```suggestion
<Tooltip
                            className={classes.tooltip}
                            title={
                                <FormattedMessage
                                    id={'securityAnalysis.toolTip'}
                                />
                            }
                            placement="left-start"
                        >
                        <InfoIcon />
                        </Tooltip>
```

Reply:
Does it interpret html? because **securityAnalysis.toolTip** contains **</br>**
I will give it a try

Reply:
Done